scripts/install.sh

```diff
@@ -22,21 +22,7 @@ fi
 
 if command_exists socketplane; then
     echo >&2 'Warning: "socketplane" command appears to already exist.'
-    while true; do
-        read -p "Would you like to re-install socketplane (y/n) " yn
-        case $yn in
-            [Yy]* )
-                cleanup
-                break
-                ;;
-            [Nn]* )
-                exit 1
-                ;;
-            * )
-                echo "Please answer yes or no."
-                ;;
-        esac
-    done
+    echo >&2 'Warning: "Please run "socketplane uninstall first'
```

Comment:
@mavenugo wanted this in here iirc. There was an issue raised a while back. WIll see if I can dig up a reference. I'm perfectly fine with erroring out at this point.